### PR TITLE
Added invalid nickname error for conversations

### PR DESCRIPTION
**Pull request by Jxpxn**
We currently cannot know if the input nickname is invalid (less than 1 character & more than 50 characters) and if the lib changed it to the default nickname ("Host").

### CODE REVIEW

**Comment by Jxpxn** [comment]
> @Jxpxn please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

File: src/sdk/Transcription/Conversation.ts [hunk 1]
```diff
@@ -202,9 +202,11 @@ export class ConversationImpl extends Conversation implements IDisposable {
             // TODO: specify the regex required. Nicknames must be unique or get the duplicate nickname error
             // TODO: check what the max length is and if a truncation is required or if the service handles it without an error
             let hostNickname: string = speechConfig.getProperty(PropertyId[PropertyId.ConversationTranslator_Name]);
-            if (hostNickname === undefined || hostNickname === null || hostNickname.length <= 1 || hostNickname.length > 50) {
+            if (hostNickname === undefined || hostNickname === null) {
                 hostNickname = "Host";
             }
+            Contracts.throwIfNullOrTooLong(hostNickname, 'nickname', 50);
```
Comment (glharper) [inline src/sdk/Transcription/Conversation.ts:208]: For this line and one below, use double quotes, otherwise this fails linting

Comment (Jxpxn) [reply src/sdk/Transcription/Conversation.ts]: I fixed it

**Review by glharper — APPROVED** [review]

**Comment by glharper** [comment]
Thanks again for this PR. :-)

**Pull request merged**